src/components/Seo/Seo.js

```diff
@@ -8,7 +8,7 @@ const Seo = props => {
   const postTitle = ((data || {}).frontmatter || {}).title;
   const postDescription = ((data || {}).frontmatter || {}).description;
   const postCover = ((data || {}).frontmatter || {}).cover;
-  const postSlug = ((data || {}).fields || {}).slug;
+  const postSlug = ((data || {}).fields || {}).slugi || '';
```

Comment:
The i added after slug is a typo or was intended?

Reply:
I noticed the same problem as well as an og:image problem and fixed with the following changes to Seo.js:

```
  const image = postCover
    ? `${config.siteUrl}${postCover.childImageSharp.resize.src}`
    : config.siteImage;
  const url = postSlug ? `${config.siteUrl}${postSlug}` : config.siteUrl;
```